src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if UNIX
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+using System.Net;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+using System.Security.Cryptography;
+using System.Security.Permissions;
+using System.Text;
+using System.Threading;
+using System.Linq;
+using Dbg = System.Management.Automation;
+
+namespace Microsoft.PowerShell.Commands
+{
+    #region Stop-Computer
+
+    /// <summary>
+    /// Cmdlet to stop computer.
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Stop, "Computer", SupportsShouldProcess = true,
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=135263", RemotingCapability = RemotingCapability.SupportedByCommand)]
+    public sealed class StopComputerCommand : PSCmdlet, IDisposable
+    {
+        #region Private Members
+
+        private Process _process = null;
+
+        #endregion
+
+        #region "Parameters"
+
+        /// <summary>
+        /// The authentication options for CIM_WSMan connection.
+        /// </summary>
+        [Parameter]
+        [ValidateSet(
+            "Default",
+            "Basic",
+            "Negotiate", // can be used with and without credential (without -> PSRP mapped to NegotiateWithImplicitCredential)
+            "CredSSP",
+            "Digest",
+            "Kerberos")] // can be used with explicit or implicit credential
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public string WsmanAuthentication { get; set; } = "Default";
```

Comment:
This parameter is not valid for UNIX and should be removed

Reply:
Removed.